src/plugins/global-settings/colors/editor.scss

```diff
@@ -61,3 +62,6 @@
 	}
 }
 
+.components-circular-option-picker__option-wrapper {
```

Comment:
This selector might also affect other non-stackable color options. Add a parent selector to ensure only Stackable controls will be affected.